code/game/area/areas/mining.dm

```diff
@@ -147,50 +145,50 @@
 	name = "Icemoon"
 	icon_state = "explored"
 	always_unpowered = TRUE
-	poweralm = FALSE
 	power_environ = FALSE
 	power_equip = FALSE
 	power_light = FALSE
 	requires_power = TRUE
 	ambientsounds = MINING
+	area_flags = UNIQUE_AREA | FLORA_ALLOWED | NO_ALERTS
 
 /area/icemoon/surface/outdoors // weather happens here
 	name = "Icemoon Wastes"
 	outdoors = TRUE
 
 /area/icemoon/surface/outdoors/labor_camp
 	name = "Icemoon Labor Camp"
-	area_flags = UNIQUE_AREA
+	area_flags = UNIQUE_AREA | NO_ALERTS
 
 /area/icemoon/surface/outdoors/unexplored //monsters and ruins spawn here
 	icon_state = "unexplored"
-	area_flags = UNIQUE_AREA | FLORA_ALLOWED | MOB_SPAWN_ALLOWED | TUNNELS_ALLOWED
+	area_flags = UNIQUE_AREA | FLORA_ALLOWED | MOB_SPAWN_ALLOWED | TUNNELS_ALLOWED | NO_ALERTS
 
 /area/icemoon/surface/outdoors/unexplored/rivers // rivers spawn here
 	icon_state = "danger"
 
 /area/icemoon/surface/outdoors/unexplored/rivers/no_monsters
-	area_flags = UNIQUE_AREA | FLORA_ALLOWED | TUNNELS_ALLOWED
+	area_flags = UNIQUE_AREA | FLORA_ALLOWED | TUNNELS_ALLOWED | NO_ALERTS
 
 /area/icemoon/underground
 	name = "Icemoon Caves"
 	outdoors = TRUE
 	always_unpowered = TRUE
 	requires_power = TRUE
-	poweralm = FALSE
 	power_environ = FALSE
 	power_equip = FALSE
 	power_light = FALSE
 	ambientsounds = MINING
+	area_flags = UNIQUE_AREA | FLORA_ALLOWED | NO_ALERTS
```

Comment:
inherited from parent so not needed

Reply:
`/area/icemoon` does not have NO_ALERTS. `/area/icemoon/surface` and `/area/icemoon/underground` do.